Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp

```diff
@@ -93,7 +98,17 @@ static WebCore::IntRect screenRectOfContents(WebCore::Element& element)
     auto viewportRect = snappedIntRect(frameView->layoutViewportRect());
     contentsRect.intersect(viewportRect);
 
-    return frameView->contentsToScreen(contentsRect);
+    return type == CoordinateConversionType::Screen ? frameView->contentsToScreen(contentsRect) : frameView->contentsToRootView(contentsRect);
+}
+
+static WebCore::IntRect screenRectOfContents(WebCore::Element& element)
```

Comment:
I think when this PR is done we won't be doing any conversions to screen coordinates in this file.